oozie/src/test/java/org/apache/falcon/oozie/process/OozieProcessWorkflowBuilderTest.java

```diff
@@ -577,8 +597,8 @@ public void testProcessMapperForTableStorage(String secureOption) throws Excepti
         assertHCatCredentials(parentWorkflow, wfPath);
     }
 
-    private Map<String, String> getExpectedProperties(Feed inFeed, Feed outFeed,
-                                                      Process process) throws FalconException {
+    private Map<String, String> getExpectedProperties(Feed inFeed, Feed outFeed, Process process)
+            throws FalconException {
```

Comment:
I don't see a checkstyle violation in the [latest build of this Pull Request](https://builds.apache.org/job/Apache-Falcon-Pull-Request-Build/55/). Are you talking specifically about line 600 that wraps to 601? The line would be 122 characters if it were not wrapped. Violating the Line Length rule. So the formatter corrects by wrapping and indenting 4 spaces inward. I think this is appropriate.


I also don't see any using Eclipse-CS with the checkstyle.xml found in the addons projects. However I am beginning to loose faith in the Eclipse-CS plug-in!